.github/workflows/require_label.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Pull Request Labels
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mheap/github-action-required-labels@v3
+        with:
+          mode: minimum
+          count: 1
+          labels: "documentation, bug, bugfix, chore, dependencies, enhancement, feature, gh-actions, ignore-for-release, security, sorbet"
```

Comment:
I took this list from all the labels available, and removed the ones that seem irrelevant for the release notes.

Reply:
See also: https://github.com/Shopify/tapioca/blob/main/.github/release.yml